### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by patrick-tolosa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ### Hey there!
 
 This repository is a work in in progress in a pre-alpha state.
```
Comment (carlevison) [inline README.md:3]:
```suggestion
This repository is a work in progress in a pre-alpha state.
```

File: README.md [hunk 2]
```diff
@@ -17,85 +17,47 @@ Feel free to visit our more mature SDKs:
...
-  .addAction(
-    scale()
-      .height(100)
-      .width(100)
-  )
-  .setConfig(configOptions)
 
-let url = imgtImage.getURLForImage('cat.jpg')
+### Key Features
+- ESM packages, import only what you need.
```
Comment (carlevison) [inline README.md:33]:
```suggestion
- ESM packages, import only what you need
```

File: README.md [hunk 3]
```diff
@@ -17,85 +17,47 @@ Feel free to visit our more mature SDKs:
...
-    
-- [ ] Classes/Concepts
-    - [ ] TransformableImage
-    
-- [ ] Open issues
-    - [ ] JSDoc and typescript have missing features (like extending objects)
+### Progress report
+We're currently working on implementing all the wide range of features and possible transformations.  
+
+We've created a progress report that visualizes our efforts
```
Comment (carlevison) [inline README.md:61]:
```suggestion
We've created a progress report that visualizes our efforts: 
```

File: README.md (hunk 1 above)
Comment (carlevison) [inline README.md:3]:
```suggestion
This repository is a work in progress in a pre-alpha state.
```

File: README.md (hunk 2 above)
Comment (carlevison) [inline README.md:33]:
```suggestion
- ESM packages, import only what you need
```

File: README.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ### Hey there!
 
 This repository is a work in in progress in a pre-alpha state.
-Needless to say it's not stable.
+Needless to say it's not stable, all APIs and Syntax are subject to change.
```
Comment (carlevison) [inline README.md:4]:
```suggestion
Needless to say it's not stable, all APIs and syntax are subject to change.
```

File: README.md [hunk 5]
```diff
@@ -17,85 +17,47 @@ Feel free to visit our more mature SDKs:
 
   
   
+----
 
 
-----
+### What is this project for?
+This project comes to replace and modernize our existing JS offering, and it's a part of a larger effort in Cloudinary.
```
Comment (strausr) [inline README.md:24]: This project comes to replace and modernize our existing JS offering, and it's part of a larger effort in Cloudinary.

File: README.md [hunk 6]
```diff
@@ -17,85 +17,47 @@ Feel free to visit our more mature SDKs:
...
   
+----
 
 
-----
+### What is this project for?
+This project comes to replace and modernize our existing JS offering, and it's a part of a larger effort in Cloudinary. 
 
-Below is the progress made on the project, the list will be modified and extended as the project progresses
+You can read more here:
```
Comment (strausr) [inline README.md:26]: You can read more about the project here:

**Pull request merged**